StockCount/View/EditViewController.swift

```diff
@@ -114,7 +161,7 @@ class EditViewController: CustomNavBarViewController, UITextViewDelegate
     }
     @objc func onBtnShare() {
         editController.saveFile(fileName, strContent: txtView.text)
-        if let fileURL = FileMgr.getURL(fileName: fileName) {
+        if let fileURL = FileMgr.saveShareFile(fileName: fileName) {
```

Comment:
What does happen if the `saveShareFile()` returns `nil` ? Should we display an error to the user?

Reply:
added to show alert for nil return